style.css

```diff
@@ -7,26 +7,23 @@ body {
   height: 100vh;
 }
 
-.main {
-  background-image: url('./imagees/mobile.png');
-  background-position: center;
-  background-size: cover;
-  height: 100vh;
-}
-
 header {
   margin-top: 2rem;
   display: flex;
   justify-content: space-between;
 }
 
-.sp1 {
-  color: blue;
-  font-size: 45px;
+a > .sp1 {
+  width: 76px;
+  height: 20px;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 20px;
+  color: #6070ff;
 }
 
 .sp2 img {
-  margin-top: 2rem;
+  margin-top: 1rem;
```

Comment:
- [ Optional ] As stated the project requirement and suggested by the previous reviewer You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly adjust the **img** margins and alignment of the **hamburger menu** just like the figma design.

Your Design Look,likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/205247018-96f51448-1a2a-41ab-a5f6-2df98f494793.PNG)

The Figma design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/205247291-f4ef9dc8-21d5-4441-8dcd-808526bb2427.PNG)